moviepy/video/io/ffmpeg_reader.py

```diff
@@ -779,7 +780,11 @@ def parse_duration(self, line):
                 r"([0-9][0-9]:[0-9][0-9]:[0-9][0-9].[0-9][0-9])",
                 time_raw_string,
             )
+            if match_duration is None:
+                raise VideoCorruptedError("Video is corrupted and missing duration")
```

Comment:
Could be useful to show what fails:
```suggestion
                raise VideoCorruptedError(f"Could not parse duration from {time_raw_string!r}")
```